tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -222,16 +222,21 @@ def test_UlyanovEtAl2016ConvSequence(subtests):
 
 
 def test_UlyanovEtAl2016JoinBlock(subtests, input_image):
-    in_channels = extract_num_channels(input_image)
+    input_image1 = input_image
+    input_image2 = torch.cat((input_image, input_image), 1)
```

Comment:
You could use `batch_up_image`

Reply:
But the `batch_size` should not change here. The difference here is that the pyramid increases the `num_channels` with each level and enters the join_block as `shallow_branch`. For this reason the num_channels must be increased and not the batch_size.

Reply:
Oops, you are right about that.